### PR TITLE
fix for c4march-20 lime-290

### PR DESCRIPTION
**Pull request by KushGoyal**
https://linear.app/sublime-finance/issue/LIME-290/20-expected-withdrawn-interest-shares-might-not-match-actual-amount

### CODE REVIEW

**Comment by prateekreddy** [comment]
@KushGoyal We can probably remove _interestToWithdraw as one of the return values as it is not being used in this function.

File: contracts/PooledCreditLine/LenderPool.sol [hunk 1]
```diff
@@ -393,17 +393,18 @@ contract LenderPool is ERC1155Upgradeable, ReentrancyGuard, IPooledCreditLineEnu
         address _strategy = pooledCLConstants[_id].borrowAssetStrategy;
         address _borrowAsset = pooledCLConstants[_id].borrowAsset;
 
-        (uint256 _interestToWithdraw, uint256 _interestSharesToWithdraw) = _calculateInterestToWithdraw(
+        (, uint256 _interestSharesToWithdraw) = _calculateInterestToWithdraw(
```
Comment (prateekreddy) [inline contracts/PooledCreditLine/LenderPool.sol:396]: We can remove the return value from the definition of _calculateInterestToWithdraw ?

Comment (KushGoyal) [reply contracts/PooledCreditLine/LenderPool.sol:396]: this is being used in another place with both the values
https://github.com/sublime-finance/sublime-v1/blob/d75a260ab2b6c1ce46661ff5c9ad982427fea2dd/contracts/PooledCreditLine/LenderPool.sol#L543

Comment (prateekreddy) [reply contracts/PooledCreditLine/LenderPool.sol:396]: right, but we can seperate that out to save gas here.

**Pull request merged**